examples/rpc-server/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "build": "lb-tsc",
     "build:watch": "lb-tsc --watch",
-    "clean": "lb-clean dist",
+    "clean": "lb-clean *example-rcp-server-*.tgz dist tsconfig.build.tsbuildinfo package",
```

Comment:
just curious, when use `*.tsbuildinfo` and when use `tsconfig.build.tsbuildinfo`? 
quickly checked benchmark package seems it only has one tsbuildinfo file

Reply:
Either way is fine.